### PR TITLE
Remove unused rocksdb.backup_dir

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

**Comment by UncP** [comment]
LGTM

File: src/config.rs [hunk 1]
```diff
@@ -356,7 +355,6 @@ pub struct DbConfig {
     pub writable_file_max_buffer_size: ReadableSize,
     pub use_direct_io_for_flush_and_compaction: bool,
     pub enable_pipelined_write: bool,
-    pub backup_dir: String,
```
Comment (BusyJay) [inline src/config.rs]: Is it backward compatible? If `backup_dir` is configured, can the configuration file be parsed successfully?

Comment (huachaohuang) [reply src/config.rs]: Yes, unspecified configs will be ignored.

**Comment by huachaohuang** [comment]
Please approve and merge this.

**Pull request merged**